packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8345,6 +8346,9 @@ export class Editor extends EventEmitter<TLEventMap> {
 	/** @internal */
 	private _selectedShapeIdsAtPointerDown: TLShapeId[] = []
 
+	/** @internal */
+	private _longPressTimeout = -1 as any
```

Comment:
you can type this as `NodeJS.Timeout | null`
`private _longPressTimeout: NodeJS.Timeout | null = null`

Reply:
I've seen this different ways (i.e. ReturnType<typeof setTimeout>) as it's a little different in different contexts. Maybe we should have a utility type for this?